ynr/apps/popolo/models.py

```diff
@@ -744,6 +745,20 @@ class Organization(Dateframeable, Timestampable, models.Model):
         "Source", help_text="URLs to source documents about the organization"
     )
 
+    # Copied from OrganizationExtra
+    slug = models.CharField(max_length=256, blank=True, unique=True)
+    register = models.CharField(blank=True, max_length=512)
+    images = GenericRelation("images.Image")
+
+    def ec_id(self):
```

Comment:
I know this isn't new code in this PR, but shouldn't this method have a special case for when I call it and the classification is not "Party" (e.g: classification = "local-authority")

Reply:
Fixed in 5223e1f